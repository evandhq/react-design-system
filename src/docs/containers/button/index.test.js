import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';

import ButtonPage from './index';

describe('snapshot testing for all examples on doc page', () => {
  it('mainButton', () => {
    const wrapper = render(<ButtonPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
