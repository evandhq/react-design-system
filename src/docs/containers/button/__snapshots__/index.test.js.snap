// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing for all examples on doc page mainButton 1`] = `
Array [
  <h1
    class="sc-bdVaJa irmrqS"
    data-test="h1-tag"
  >
    دکمه ها
  </h1>,
  <br />,
  <br />,
  <h2
    class="sc-bwzfXH hlvhiK"
    data-test="h2-tag"
  >
    نمونه دکمه پیشفرض
  </h2>,
  <button
    class="sc-kAzzGY gLJdoB"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <br />,
  <h2
    class="sc-bwzfXH hlvhiK"
    data-test="h2-tag"
  >
    دکمه آیکون دار
  </h2>,
  <button
    class="sc-kAzzGY gLJdoB"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <div
      class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
    >
      <i
        class="icon-watch_later sc-jzJRlG gBuJEN"
        color="white"
        data-test="icon"
      />
    </div>
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه آیکون دار
    </span>
  </button>,
  <br />,
  <h2
    class="sc-bwzfXH hlvhiK"
    data-test="h2-tag"
  >
    دکمه full-width
  </h2>,
  <button
    class="sc-kAzzGY jgoHpL"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه full-width (sm size)
    </span>
  </button>,
  <button
    class="sc-kAzzGY criAQW"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه full-width(md size - default)
    </span>
  </button>,
  <button
    class="sc-kAzzGY iuoOjW"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه full-width(lg size)
    </span>
  </button>,
  <br />,
  <h2
    class="sc-bwzfXH hlvhiK"
    data-test="h2-tag"
  >
    دکمه در اندازه های متفاوت
  </h2>,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    sm size
  </h3>,
  <button
    class="sc-kAzzGY cuuIPS"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <button
    class="sc-kAzzGY cuuIPS"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <div
      class="sc-chPdSV kEYvSi sc-cSHVUG ebkVMz"
    >
      <i
        class="icon-watch_later sc-jzJRlG kMIXPq"
        color="white"
        data-test="icon"
      />
    </div>
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <br />,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    md size (default)
  </h3>,
  <button
    class="sc-kAzzGY gLJdoB"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <button
    class="sc-kAzzGY gLJdoB"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <div
      class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
    >
      <i
        class="icon-watch_later sc-jzJRlG gBuJEN"
        color="white"
        data-test="icon"
      />
    </div>
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <br />,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    lg size
  </h3>,
  <button
    class="sc-kAzzGY jRwfLb"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <button
    class="sc-kAzzGY jRwfLb"
    data-styletype="primary"
    data-test="button"
    type="submit"
  >
    <div
      class="sc-chPdSV kEYvSi sc-cSHVUG hKScKU"
    >
      <i
        class="icon-watch_later sc-jzJRlG cGQQZc"
        color="white"
        data-test="icon"
      />
    </div>
    <span
      class="sc-htoDjs bgiUUZ"
      color="white"
      data-test="span"
    >
      دکمه
    </span>
  </button>,
  <br />,
  <h2
    class="sc-bwzfXH hlvhiK"
    data-test="h2-tag"
  >
    دکمه ها در استایل ها متفاوت و رنگ های متفاوت
  </h2>,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    رنگ قرمز
  </h3>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ قرمز - normal
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY iWxklW"
      data-styletype="primary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default)
      </span>
    </button>
    <button
      class="sc-kAzzGY iWxklW"
      data-styletype="secondary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG kexrok"
          color="red"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs Bjkai"
        color="red"
        data-test="span"
      >
        دکمه secondary
      </span>
    </button>
    <button
      class="sc-kAzzGY iWxklW"
      data-styletype="tertiary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary
      </span>
    </button>
  </div>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ قرمز - disabled
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY fYrDtw"
      data-styletype="primary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default) - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY fYrDtw"
      data-styletype="secondary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG kexrok"
          color="red"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs Bjkai"
        color="red"
        data-test="span"
      >
        دکمه secondary - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY fYrDtw"
      data-styletype="tertiary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary - disabled
      </span>
    </button>
  </div>,
  <hr />,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    رنگ سبز
  </h3>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ سبز - normal
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY dwduYj"
      data-styletype="primary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default)
      </span>
    </button>
    <button
      class="sc-kAzzGY dwduYj"
      data-styletype="secondary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG ieIRpH"
          color="green"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs jpsEjs"
        color="green"
        data-test="span"
      >
        دکمه secondary
      </span>
    </button>
    <button
      class="sc-kAzzGY dwduYj"
      data-styletype="tertiary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary
      </span>
    </button>
  </div>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ سبز - disabled
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY bhgDad"
      data-styletype="primary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default) - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY bhgDad"
      data-styletype="secondary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG ieIRpH"
          color="green"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs jpsEjs"
        color="green"
        data-test="span"
      >
        دکمه secondary - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY bhgDad"
      data-styletype="tertiary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary - disabled
      </span>
    </button>
  </div>,
  <hr />,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    رنگ زرد
  </h3>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ زرد - normal
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY gHRNcx"
      data-styletype="primary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default)
      </span>
    </button>
    <button
      class="sc-kAzzGY gHRNcx"
      data-styletype="secondary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG iWLfHZ"
          color="yellow"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs jdlLkZ"
        color="yellow"
        data-test="span"
      >
        دکمه secondary
      </span>
    </button>
    <button
      class="sc-kAzzGY gHRNcx"
      data-styletype="tertiary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary
      </span>
    </button>
  </div>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ زرد - disabled
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY hDvnjn"
      data-styletype="primary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default) - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY hDvnjn"
      data-styletype="secondary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG iWLfHZ"
          color="yellow"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs jdlLkZ"
        color="yellow"
        data-test="span"
      >
        دکمه secondary - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY hDvnjn"
      data-styletype="tertiary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary - disabled
      </span>
    </button>
  </div>,
  <hr />,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    رنگ آبی
  </h3>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ آبی - normal
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY jhFpFm"
      data-styletype="primary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default)
      </span>
    </button>
    <button
      class="sc-kAzzGY jhFpFm"
      data-styletype="secondary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gjBRpW"
          color="blue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs kavVik"
        color="blue"
        data-test="span"
      >
        دکمه secondary
      </span>
    </button>
    <button
      class="sc-kAzzGY jhFpFm"
      data-styletype="tertiary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary
      </span>
    </button>
  </div>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ آبی - disabled
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY kcudCe"
      data-styletype="primary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default) - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY kcudCe"
      data-styletype="secondary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gjBRpW"
          color="blue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs kavVik"
        color="blue"
        data-test="span"
      >
        دکمه secondary - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY kcudCe"
      data-styletype="tertiary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary - disabled
      </span>
    </button>
  </div>,
  <hr />,
  <h3
    class="sc-htpNat djFCwm"
    data-test="h3-tag"
  >
    رنگ آبی تیره - پیشفرض
  </h3>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ آبی تیره - پیشفرض - normal
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY gLJdoB"
      data-styletype="primary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default)
      </span>
    </button>
    <button
      class="sc-kAzzGY gLJdoB"
      data-styletype="secondary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه secondary
      </span>
    </button>
    <button
      class="sc-kAzzGY gLJdoB"
      data-styletype="tertiary"
      data-test="button"
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary
      </span>
    </button>
  </div>,
  <h4
    class="sc-bxivhb Pslhp"
    data-test="h4-tag"
  >
    رنگ آبی تیره - پیشفرض - disabled
  </h4>,
  <div
    class="sc-kgoBCf beOYFC"
  >
    <button
      class="sc-kAzzGY jhkFYr"
      data-styletype="primary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG gBuJEN"
          color="white"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs bgiUUZ"
        color="white"
        data-test="span"
      >
        دکمه primary (default) - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY jhkFYr"
      data-styletype="secondary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه secondary - disabled
      </span>
    </button>
    <button
      class="sc-kAzzGY jhkFYr"
      data-styletype="tertiary"
      data-test="button"
      disabled=""
      type="submit"
    >
      <div
        class="sc-chPdSV kEYvSi sc-cSHVUG hDoLhH"
      >
        <i
          class="icon-watch_later sc-jzJRlG cMJMwC"
          color="darkBlue"
          data-test="icon"
        />
      </div>
      <span
        class="sc-htoDjs iQeslt"
        color="darkBlue"
        data-test="span"
      >
        دکمه tertiary - disabled
      </span>
    </button>
  </div>,
  <br />,
]
`;
